pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -682,6 +682,16 @@ private static int GetWaitCycles(int millis = WAIT_TIME)
             return  (millis * waitMultiplier) / SLEEP_INTERVAL; // Return the wait cycle count
         }
 
+        /// <summary>
+        /// Convenience function for getting a value on the UI thread
+        /// </summary>
+        public static T GetUIValue<T>(Func<T> act)
```

Comment:
I would have probably named this "RunUI" (or maybe "CallUI" if "RunUI" caused too much compiler confusion)
This method does the same thing as "RunUI", but has a Func parameter instead Action.